docs/app/pages/components/components-sections/input-controls/typeahead/snippets/app.html

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+    <div class="col-md-12 has-feedback">
+        <form class="input-typeahead-form">
+            <input [value]="value"
+                   [(ngModel)]="input"
+                   [ngModelOptions]="{standalone: true}"
+                   [placeholder]="'Enter Text'"
+                   class="form-control"
+                   (keydown.enter)="dropdownOpen = true; $event.preventDefault()"
+                   (click)="dropdownOpen = true; $event.stopPropagation();"
```

Comment:
Do we need these `stopPropgation` and `preventDefault` calls? We might do, taking them out I didn't see any difference but I could be wrong